auction-server/src/opportunity/entities/opportunity_svm.rs

```diff
@@ -33,14 +33,14 @@ pub struct OpportunitySvmProgramLimo {
 
 #[derive(Debug, Clone, PartialEq)]
 pub struct OpportunitySvmProgramWallet {
-    pub user_wallet_address:         Pubkey,
-    pub maximum_slippage_percentage: f64,
+    pub user_wallet_address:  Pubkey,
+    pub maximum_slippage_bps: u16,
 }
 
 #[derive(Debug, Clone, PartialEq)]
 pub enum OpportunitySvmProgram {
     Limo(OpportunitySvmProgramLimo),
-    Phantom(OpportunitySvmProgramWallet),
+    Swap(OpportunitySvmProgramWallet),
```

Comment:
I think we need a way to extract the program for each of these requests. Are we going to use router account for it? :-?